lib/src/features/archive/presentation/widgets/archive_populated.dart

```diff
@@ -17,51 +18,62 @@ class ArchivePopulated extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    SystemChrome.setApplicationSwitcherDescription(
```

Comment:
Does this also work with other browsers?

Reply:
yes, atleast with all who has based on chrome. I will try it with safari.